### PR TITLE
Fix breaking tests

### PR DESCRIPTION
**Pull request by lirantal**
## Description

Fix breaking E2E tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Tests
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: __tests__/__e2e__/cli.e2e.js [hunk 1]
```diff
@@ -22,23 +22,28 @@ describe('End-to-End CLI', () => {
       })
     } catch (err) {
       expect(err.code).toBe(2) // means that vulnerabilities were found
+      console.log(err)
+      console.log(err && err.stdout)
+      console.log(err && err.stderr)
```
Comment (dkorn) [inline __tests__/__e2e__/cli.e2e.js]: Are you logging the err details for future tests debugging?

File: __tests__/__e2e__/cli.e2e.js [hunk 2]
```diff
@@ -22,23 +22,28 @@ describe('End-to-End CLI', () => {
...
     }
   })
 
   test('CLI should show vulnerabilities breakdown numbers and their titles', async () => {
-    expect.assertions(8)
+    expect.assertions(9)
```
Comment (dkorn) [inline __tests__/__e2e__/cli.e2e.js]: Nice catch!

In general, do you really want to bump this number with every time we add more info to the output?

File: __tests__/__e2e__/cli.e2e.js [hunk 3]
```diff
@@ -63,14 +71,17 @@ describe('End-to-End CLI', () => {
...
+    console.log(err && err.stderr)
+    console.log(stdout)
     expect(err).toBe(undefined)
     expect(stdout).toContain('0 vulnerabilities found')
   })
 
   test('CLI should accept path to project directory from command argument', async () => {
     const project3Dir = path.join(__dirname, 'project3')
-    const { stdout, err } = await spawnAsync('node', [cliBinPath, `--directory=${project3Dir}`], {
+    const { stdout, err } = await spawnAsync('node', [cliBinPath, '--directory', project3Dir], {
```
Comment (dkorn) [inline __tests__/__e2e__/cli.e2e.js]: 👍

**Comment by lirantal** [comment]
Unfortunately I can't re-start the Windows tests to figure out why it fails or confirm it: 
image

File: __tests__/__e2e__/cli.e2e.js (hunk 1 above)
Comment (lirantal) [reply __tests__/__e2e__/cli.e2e.js]: No, just for this run to figure out why the windows PR times out

File: __tests__/__e2e__/cli.e2e.js (hunk 2 above)
Comment (lirantal) [reply __tests__/__e2e__/cli.e2e.js]: I don't.
I think we can change to just has.assertions() as the generic API

**Comment by lirantal** [comment]
Woohoo, finally got us some test coverage for E2E Windows too! 🎉

**Pull request merged**